bin/cache_daemon.ml

```diff
@@ -123,9 +128,13 @@ let term =
          value
          & opt (some int) None
          & info ~docv:"BYTES" [ "size" ] ~doc:"size to trim the cache to")
+     and+ quiet =
+       let doc = "Only output the daemon endpoint" in
+       Arg.(value & flag & info [ "quiet" ] ~doc)
```

Comment:
Is it possible to follow dune's convention a little more here? E.g. be quiet by default and have a `--display` option to control verbosity with descriptive modes

Reply:
Sure ; would a `--display script` be fine ? To explicit we want a clean, easily parsed output.

Reply:
Also I think `dune cache start` should by default output "something", otherwise it's going to always return right away silently, which seems a bit off. But I'll go with the consensus choice.

Reply:
Well, if we want to signal success I would just make sure we get the exit codes right (which we do). But to be honest, I don't see a huge deal if you output something. We can always change it later if people complain about the noise.

Reply:
It's now silent by default and prints out the endpoint in any display mode but silent.